lib/GitHub.js

```diff
@@ -19,7 +19,12 @@ class GitHub extends GitService {
       const isAppAuth = config.get('githubAppID') &&
         config.get('githubPrivateKey')
       const isLegacyAuth = config.get('githubToken') &&
-        ['1', '2'].includes(options.version)
+        /*
+         * Allowing/requiring the Staticman version to bleed-through to here is problematic, as
+         * it may not always be available to the caller (e.g., controllers/handlePR.js). To allow
+         * for this, we allow for a blank version value.
+         */
+        ['1', '2', ''].includes(options.version)
```

Comment:
```suggestion
        // permit token auth when no API version is provided
        ['', '1', '2'].includes(options.version)
```

nit: can be less verbose here without losing the important message we seek to convey

Reply:
Changed as requested.